class-tgm-plugin-activation.php

```diff
@@ -2272,7 +2278,8 @@ protected function get_plugin_status_text( $slug ) {
 			}
 
 			return sprintf(
-				_x( '%1$s, %2$s', '%1$s = install status, %2$s = update status', 'tgmpa' ),
+				/* translators: %1$s: install status, %2$s: update status */
+				__( '%1$s, %2$s', 'tgmpa' ),
 				$install_status,
```

Comment:
@jrfnl,

The **context** is used to describe the entire string, **translators comment** are used to describe specific placeholders like `%s`, `%1$s`, `%2$s` and others.

![Translators comment location](http://i.imgur.com/xDU1lkL.png)

In this case, translators comment are needed because we have two placeholders in this string.

As for the context, I'm not sure. We can set "**Install/Update Status**" as a context, but It's not providing more information than the translator comment.